themes/default/views/admin/orders/show.blade.php

```diff
@@ -2,249 +2,282 @@
+                                    </div>
+                                </div>
+                            @endif
+
+                            <div>
+                                <form> <!-- TODO: DELETE ONLY THIS SERVICE, NOT ORDER route('admin.orders.delete', $order->id) -->
+                                    <button class="button button-danger" data-tooltip-target="tooltip-animation7">
+                                        <i class="ri-delete-bin-2-line"></i>
+                                    </button>
+                                </form>
+                                <div id="tooltip-animation7" role="tooltip" class="absolute z-10 invisible inline-block px-3 py-2 text-sm font-medium text-white transition-opacity duration-300 bg-gray-900 rounded-lg shadow-sm opacity-0 tooltip dark:bg-gray-700">
+                                    TODO: DELETE ONLY THIS SERVICE, NOT ORDER
+                                    <div class="tooltip-arrow" data-popper-arrow></div>
+                                </div>
+                            </div>
+                        </td>
+
+                    </tr>
+                @endforeach
+                </tbody>
+            </table>
+        </div>
+    </div>
+
+    <!-- Change quantity / price modal -->
+    <div id="changePriceQuantity{{ $product->id }}" tabindex="-1" aria-hidden="true"
+         class="fixed top-0 left-0 right-0 z-50 hidden w-full p-4 overflow-x-hidden overflow-y-auto md:inset-0 h-[calc(100%-1rem)] md:h-full">
+        <div class="relative w-full h-full max-w-2xl md:h-auto">
+            <div class="relative bg-white rounded-lg shadow dark:bg-gray-700">
+                <div
+                    class="flex items-start justify-between p-4 border-b rounded-t dark:border-gray-600">
+                    <h3 class="text-xl font-semibold text-gray-900 dark:text-white">
+                        {{ __('Change Price/Quantity') }}
+                    </h3>
+                    <button type="button"
+                            class="text-gray-400 bg-transparent hover:bg-gray-200 hover:text-gray-900 rounded-lg text-sm p-1.5 ml-auto inline-flex items-center dark:hover:bg-gray-600 dark:hover:text-white"
+                            data-modal-hide="changePriceQuantity{{ $product->id }}">
+                        <svg aria-hidden="true" class="w-5 h-5" fill="currentColor"
+                             viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg">
+                            <path fill-rule="evenodd"
+                                  d="M4.293 4.293a1 1 0 011.414 0L10 8.586l4.293-4.293a1 1 0 111.414 1.414L11.414 10l4.293 4.293a1 1 0 01-1.414 1.414L10 11.414l-4.293 4.293a1 1 0 01-1.414-1.414L8.586 10 4.293 5.707a1 1 0 010-1.414z"
+                                  clip-rule="evenodd"></path>
+                        </svg>
+                        <span class="sr-only">Zamknij</span>
```

Comment:
Use a language variable instead of hard coding it

Reply:
Corrected, thanks